continuum-auth/README.md

```diff
@@ -0,0 +1,19 @@
+# continuum-auth
+
+**The purpose of this service is to implement an interface to connect the eLife journal to libero reviewer**
+
+A connector service so that eLife's deployment of libero can authenticate using the existing journal login system. After successful authentication, this service will redirect to an endpoint, with a
+signed jwt token appended to the URL. This token will then be used by the client to authenticate it's subsequent HTTP requests with other services in the system. This token will contain some information
+about the user, provided by the `profiles` service.
+
+## Things to define for continuum-auth to be useful long term
+
+- We need to stabilise the interface for the auth token, so that we can swap out this service with something else
+- We need some way to manually test the login flow from end to end. This means that it needs to be:
+  - Deployed somewhere - the journal auth subsystem needs to know where to redirect to
+  - Configurable - the service needs to work in many different environments, including but not limited to: locally (without docker), locally (inside docker), unstable libero environment, unstable eLife environment
+  - Working - This service is a key part of a user's entry point into the application, it provides the user with their permissions and their authentication token - it needs to be reliable!
+- We have an issue with using the `profiles` service, it does not contain information about staff members, and there is no good mechanism to map `profiles` records to records in the `people` service (the `people` service is for staff). Therefore, we need some way to map either `profiles.id` or `profiles.orcid` to some information about that user's permission, so we can calculate the role properly.
```

Comment:
it is true there is no relation between the two, one is built via ORCID-based logins and the other is maintained by staff and has a smaller scope

Reply:
That makes sense. I guess the point I was trying to make is that there's no way of using them together (without indexing all of their data, which is something I don't really want to do) for this service to calculate the user's roles